### PR TITLE
create addStyles HOC and use it in button-core.js and view.js

### PR DESCRIPTION
**Pull request by kevinbarabash**
This allows us to easily add the same `style` prop to any component we want.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 2d8b8531de6a5a9b898b2075e9641bec6e334a67

https://deploy-preview-80--wonder-blocks.netlify.com

**Comment by kevinbarabash** [comment]
I'm not sure why `npm test` didn't fail locally.  :\

File: packages/wonder-blocks-core/docs.md [hunk 1]
```diff
@@ -37,4 +37,28 @@ Other props can be passed through `View` or `Text`, as if they were normal tags.
...
-```
\ No newline at end of file
+```
+
+There's also an HOC that adds the same style prop to existing 
+components.
+
+```js
+// This should actually be `import {addStyle} from "wonder-blocks-core"`
+const {addStyle} = require("./util/add-style.js");
```
[the remark this answers is not in the document]
Comment (kevinbarabash) [reply packages/wonder-blocks-core/docs.md]: `styleguidist` doesn't know how to handle `import`s.  I'll update the comment to make things clear it's a `styleguidist` limitation not a `wonder-blocks` limitation.

File: .eslintignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
 node_modules
 coverage
 **/generated-snapshot.test.js
+
+# eslint can't handle the type defintion for the addStyle HOC
+packages/wonder-blocks-core/util/add-style.js
```
[the remark this answers is not in the document]
Comment (kevinbarabash) [reply .eslintignore]: `webapp`'s `eslint` doesn't run on `third_party` so I think we should be fine.

File: packages/wonder-blocks-core/docs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeresig) [reply packages/wonder-blocks-core/docs.md]: err, I know - I meant why didn't you write `const {addStyle} = require("wonder-blocks-core");`? Would be a bit easier to understand to other devs.

[the remark this answers is not in the document]
Comment (kevinbarabash) [reply packages/wonder-blocks-core/docs.md]: Ah... I see what you're getting at.  `node` can't find a named module from within itself.  :(

[the remark this answers is not in the document]
Comment (jeresig) [reply packages/wonder-blocks-core/docs.md]: sigh... ok. Maybe `require("./index.js")` instead? (I think we've done that elsewhere)

**Pull request merged**